core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -321,8 +321,8 @@ class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V]) extends Actor w
   def poll(): Unit = {
     val wakeupTask = context.system.scheduler.scheduleOnce(settings.wakeupTimeout) {
       log.warning(
-        "KafkaConsumer poll has exceeded wake up timeout ({}ms). Waking up consumer to avoid thread starvation.",
-        settings.wakeupTimeout.toMillis
+        "KafkaConsumer poll has exceeded wake up timeout ({}). Waking up consumer to avoid thread starvation.",
+        settings.wakeupTimeout
```

Comment:
You could call `toCoarsest` on `wakeupTimeout` here and below so it is printed as seconds or minutes instead of milliseconds.